velox/dwio/common/tests/TypeTests.cpp

```diff
@@ -55,6 +55,12 @@ TEST(TestType, selectedType) {
   EXPECT_EQ(11, cutType->maxId());
   EXPECT_EQ(2, cutType->childAt(0)->maxId());
 
+  auto origTypeChild = typeWithId->childByName("col1");
```

Comment:
Let's make this test stronger: we need to make sure all fields in original typeWithId remain unchanged, in addition to make sure the un-selected one is unchanged.